app/configurator/components/annotators.tsx

```diff
@@ -0,0 +1,187 @@
+import { Trans } from "@lingui/macro";
+import { Theme, Typography, TypographyProps } from "@mui/material";
+import { makeStyles } from "@mui/styles";
+import clsx from "clsx";
+
+import { getChartConfig } from "@/config-types";
+import {
+  ControlSection,
+  ControlSectionContent,
+  SubsectionTitle,
+} from "@/configurator/components/chart-controls/section";
+import {
+  ChartAnnotatorTabField,
+  LayoutAnnotatorTabField,
+} from "@/configurator/components/field";
+import { getFieldLabel } from "@/configurator/components/field-i18n";
+import {
+  isConfiguring,
+  isLayouting,
+  useConfiguratorState,
+} from "@/configurator/configurator-state";
+import { useLocale } from "@/locales/use-locale";
+
+const useStyles = makeStyles<Theme, { interactive?: boolean }>({
+  text: {
+    marginBottom: 4,
+    cursor: ({ interactive }) => (interactive ? "pointer" : "text"),
+
+    "&:hover": {
+      textDecoration: ({ interactive }) => (interactive ? "underline" : "none"),
+    },
+  },
```

Comment:
Nit: not very important but I personally tend not to use params for this kind of thing, but only CSS classes. So have an `interactive` class making the change, and applying it conditionally. I find that it is closer to normal CSS, thus maybe more future proof.